features/suites/normalize_relations/hydra/error.feature

```diff
@@ -0,0 +1,93 @@
+Feature: Error handling
+  In order to be able to handle error client side
+  As a client software developer
+  I need to retrieve an Hydra serialization of errors
+
+  Scenario: Get an error
+    When I send a "POST" request to "/dummies" with body:
+    """
+    {}
+    """
+    Then the response status code should be 400
+    And the response should be in JSON
+    And the header "Content-Type" should be equal to "application/ld+json"
+    And the JSON should be equal to:
+    """
+    {
+      "@context": "/contexts/ConstraintViolationList",
+      "@type": "ConstraintViolationList",
+      "hydra:title": "An error occurred",
+      "hydra:description": "name: This value should not be blank.\n",
+      "violations": [
+        {
+          "propertyPath": "name",
+          "message": "This value should not be blank."
+        }
+      ]
+    }
+    """
+
+  Scenario: Get an error during deserialization of simple relation
+    When I send a "POST" request to "/dummies" with body:
+    """
+    {
+      "name": "Foo",
+      "relatedDummy": {
+        "name": "bar"
+      }
+    }
+    """
+    Then the response status code should be 500
```

Comment:
It's a doctrine exception of type "A new entity was found through the relationship...".
I don't know how to deal with this exception. There is the same one without `denormalize_relations` to false if you don't configure cascade persist correctly.
For me it's not a problem to have & 500 here. In fact, I'm not even sure it should be under test.